.github/workflows/project_automation.yml

```diff
@@ -12,7 +12,11 @@ jobs:
   Handle_event:
     runs-on: ubuntu-latest
     steps:
-      - uses: takb/project-board@master
+      - name:
```

Comment:
Should there be a name here?

Reply:
you're right, though it really doesn't matter...